src/deadline/blender_adaptor/BlenderAdaptor/adaptor.py

```diff
@@ -294,15 +294,15 @@ def _start_blender_client(self) -> None:
             os.environ["PYTHONPATH"] = python_path_addition
 
         if self.init_data["render_engine"] == "cycles":
-            _logger.warn("Missing configuration for cycles render engine.")
-            pass
+            _logger.warning("Missing configuration for cycles render engine.")
 
         self._blender_client = LoggingSubprocess(
             args=[
                 blender_exe,
                 "--background",
                 "--python",
                 self.blender_client_path,
+                "--python-use-system-env",
```

Comment:
Is this compatible across all the supported Blender versions?

Reply:
It goes back to at least 3.0. I did a render test with 3.6 and this change.